ide/atom/README.md

```diff
@@ -0,0 +1,47 @@
+[Packages](#packages) | [Shortcuts](#shortcuts) | [CLI](#cli) | [Feedback](#feedback)
+
+# Atom
+
+Download: <https://atom.io/>
+
+## Packages
```

Comment:
Are we going to update this every time this list changes/updates? Maybe we should leave off the version numbers and/or only list the packages that we all use that aren't already installed by default?